src/js/mainPage.js

```diff
@@ -1,4 +1,8 @@
 import fillBestSellers from './fillBestSellers';
 import executeWithLoader from './service/executeWithLoader';
 
-executeWithLoader(() => fillBestSellers());
\ No newline at end of file
+fillBestSellers();
+
+executeWithLoader(() => fillBestSellers());
+
+import './selectedcategorymarkup';
```

Comment:
move "import" into top of file and insted of including js directly into this file use function that is exported from this file, this will let us use your code in different circumstances witout modifying it